lib/Doctrine/Import/Builder.php

```diff
@@ -990,6 +995,56 @@ private function innerBuildActAs($actAs, $level = 0, $parent = null, array &$emi
         return $build;
     }
 
+    /**
+     * Adds the columns of the used actAs behaviors to the comment block.
+     *
+     * @param string $className
+     * @param array $instanceOptions
+     * @param array $actAsColumns
+     *
+     * @throws Doctrine_Import_Builder_Exception
+     */
+    private function addActAsColumnsToDefinition($className, $instanceOptions, &$actAsColumns)
+    {
+        // No class specified or class does not exist.
+        if (!$className || !class_exists($className)) {
+            return;
+        }
+
+        $actAsInstance = new $className($instanceOptions);
+        $options = $actAsInstance->getOptions();
+
+        if (count($options) == 0) {
+            return;
+        }
+
+        // Some behaviors do not contain an array of columns, e.g. SoftDelete.
+        if (!is_array(reset($options))) {
+            $options = [$options];
```

Comment:
for PHP 5.3 compatibility.

```suggestion
            $options = array($options);
```

Reply:
It's time to bring in the php-cs-fixer settings. :)
I will do it.

Reply:
@connorhu Great! I'll mark this conversation as resolved, maybe adding a new issue for it?

Reply:
@thirsch done